README.md

```diff
@@ -7,59 +7,98 @@
 
 
 SKALE client tools.  
-You can use this library to develop browser and Node.js applications that use Skale.
+You can use this library to develop browser and Node.js 
+applications that use Skale.
 
 ## Installation
 
-#### NPM
+#### Node
 
 ```bash
-npm install @skalenetwork/skale-api
+npm install @skalenetwork/skale.js
 ```
 
-#### Yarn
-
-```bash
-yarn add @skalenetwork/skale-api
+## Usage
+```js
+const skale = require('@skalenetwork/skale.js');
+
+console.log(skale);
```

Comment:
consider adding contract call invocation example here

Reply:
i'm not agree, we have a lot of examples in `examples` folder

Reply:
ok, fine